tests/seeds/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v.json

```diff
@@ -1 +1 @@
-{"pubkey":"EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v","account":{"lamports":122299004116,"data":["AQAAAGBnK+E8EZLmY+NLl5Z2tJIugTZzVw70CZXoMR8Om0u0wQ1Pou7/EAAGAQEAAAAqnl7btTwEZ5CY/3sSZRcUQ0/AjFYqmjuGEQXmctQicw==","base64"],"owner":"TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA","executable":false,"rentEpoch":291}}
```

Comment:
@Taylor123 we should probably ignore this seed file and adjust the seed generation accordingly. It wasn't an issue when no one else was contributing to this repo XD